scripts/utils/update_package.py

```diff
@@ -89,12 +89,12 @@ def update_github_url(package):
     # Use findall as some packages have two urls (for 32 and 64 bits), we need to update both
     # Match urls like https://github.com/mandiant/capa/releases/download/v4.0.1/capa-v4.0.1-windows.zip
     matches = re.findall(
-        "[\"'](?P<url>https://github.com/(?P<org>[^/]+)/(?P<project>[^/]+)/releases/download/(?P<version>[^/]+)/[^\"']+)[\"']",
+        "[\"'](?P<url>https://github.com/(?P<org>[^/]+)/(?P<project>[^/]+)/releases/download/v?(?P<version>[^/]+)/[^\"']+)[\"']",
```

Comment:
I like the idea, but this change breaks updates for tools which don't include `v` in the release name, for example:
- `https://github.com/myfreeer/7z-build-nsis/releases/download/23.01/7z2301-x86.exe`
- `https://github.com/SnaffCon/Snaffler/releases/download/1.0.150/Snaffler.exe`
- `https://github.com/winsiderss/si-builds/releases/download/3.0.7645/systeminformer-3.0.7645-release-bin.zip`

what about leaving the regular expression as it was and deleting the `v` afterward if contained in the matched version? I think that should work for all cases. :thinking:

Reply:
This shouldn't be the case. Checking on online regex it looks good :
![image](https://github.com/mandiant/VM-Packages/assets/58483316/2dfa2627-15e8-4583-b5fe-baad8cda6de3). How did you find that it breaks updates ?
If it's too difficult to solve with regex, I will do as you suggest.

Reply:
Sorry, I didn't realized the `v` has a `?` which makes it optional. The change LGTM, thanks!